packages/config-schema/tests/index.test.ts

```diff
@@ -0,0 +1,33 @@
+import fs from 'fs'
+import path from 'path'
+import yaml from 'yaml'
+import Ajv from 'ajv'
+
+import draft6MetaSchema from 'ajv/dist/refs/json-schema-draft-06.json'
+
+const ajv = new Ajv()
+ajv.addMetaSchema(draft6MetaSchema)
+
+const schema = JSON.parse(fs.readFileSync('schema.json', 'utf8'))
+
+describe('test valid schemas', () => {
+  const basePath = path.join(__dirname, 'valid')
```

Comment:
I'd advise against "on-the-fly generated tests" as it might lead to unexpected issues if the generation fails silently (i.e. tests are moved and then we are not testing anything at all). It might be better if you added the expected tests over an array and iterated over them instead of generating them dynamically.